src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -146,7 +148,7 @@ public function hasProductInAssociation(string $productName, string $productAsso
 
         Assert::notNull($products);
 
-        return null !== $products->find('css', sprintf('.sylius-product-name:contains("%s")', $productName));
+        return $productName === $products->find('css', sprintf('[data-test-product-name="%s"]', $productName))->getText();
```

Comment:
You can use here `getElement` method

Reply:
Here I can't because if I use getElement and return products, next if I try use getElement on products I get error.

![Screenshot 2019-12-20 at 07 27 18](https://user-images.githubusercontent.com/39232096/71235046-464fec00-22fb-11ea-8f00-30b5b7d3786b.png)